gatsby-ssr.tsx

```diff
@@ -3,34 +3,3 @@
  *
  * See: https://www.gatsbyjs.org/docs/ssr-apis/
  */
-
```

Comment:
I think we can just remove this file. I dont think this is doing anything anymore, unless there is a reason we need to keep it around for gatsby